gitmail.rb

```diff
@@ -1,10 +1,13 @@
+require 'tmpdir'
+
 class Finder
   def initialize(repo_name)
-    `git clone https://github.com/#{repo_name}.git temp_email`
-    Dir.chdir("temp_email")
-    @emails = `git log --pretty=format:'%an %ae'`
-    Dir.chdir("..")
-    `rm -rf temp_email`
+    Dir.mktmpdir("GitMail") do |dir|
```

Comment:
Just to clarify, the tmp folder won't be called "GitMail", just be prefixed with it.

On OSX, it will be something like `/var/folders/yw/39yd7jd55bn1vw8vk4zxrqgm0000gp/T/GitMail20160315-17813-2j50fw`